src/App.tsx

```diff
@@ -36,6 +37,8 @@ const App: React.FunctionComponent = () => {
   const [second, setSecond] = useState<number | null | undefined>()
   const [isStart, setIsStart] = useState<boolean>(false)
   const [statu, setStatu] = useState<StateDef>(StateDef.ToStarted)
```

Comment:
I think `statu` is a wrong word here, we should use `status` instead, you can try google for more info. I googled because I never see ppl use `statu` in code.

Reply:
Yes, you're right. I'm always confused about naming.